README.md

```diff
@@ -67,13 +67,18 @@ const runTest = async () => {
     duration: 10000,
   };
 
-  const { writeResults } = await measurePerformance(bundleId, testCase);
+  const { writeResults } = await measurePerformance("bundleId", testCase);
   writeResults();
 };
 
 runTest();
 ```
 
+You have to replace a few elements in this script:
+- the app package
+- the app activity
+- the bundleId
```

Comment:
@Almouro why the difference in the naming? As the value was the same for the bundleId and the app package for me I'm not sure if I should let the bundleId here?

Reply:
@charlotteisambert `bundleId` and `appPackage` are the same thing. `appPackage` comes from [Appium](https://appium.io/docs/en/writing-running-appium/caps/) but tbf our wrapper could rename it and we could pass `bundleId` as well

Reply:
@charlotteisambert @Almouro `bundleId` should be replace with `driver.bundleId` since the information is already in the driver. 
`bundleId = appPackage` as defined here : https://github.com/bamlab/android-performance-profiler/blame/main/packages/appium-helper/AppiumDriver.ts#L87